yarn is required to install this library with npm

# Bug

Installing this package using `npm` throws this error:

```
npm ERR! prepareGitDep sh: yarn: command not found
npm ERR! prepareGitDep npm ERR! code ELIFECYCLE
npm ERR! prepareGitDep npm ERR! syscall spawn
npm ERR! prepareGitDep npm ERR! file sh
npm ERR! prepareGitDep npm ERR! errno ENOENT
npm ERR! prepareGitDep npm ERR! react-native-device-info@5.4.1 prepare: `yarn build`
npm ERR! prepareGitDep npm ERR! spawn ENOENT
npm ERR! prepareGitDep npm ERR!
npm ERR! prepareGitDep npm ERR! Failed at the react-native-device-info@5.4.1 prepare script.
```

It appears it expects `yarn` to be installed.  If this is expected, it would be helpful if the documentation states as such.  However, the docs state you can use either `npm` or `yarn`.

## Environment info

React native info output:

```bash
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 128.77 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    Yarn: 1.17.3 - ~/.nvm/versions/node/v10.16.0/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 23, 25, 26, 27, 28, 29
      Build Tools: 28.0.2, 28.0.3, 29.0.0
      System Images: android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-29 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.3/11C29 - /usr/bin/xcodebuild
  npmPackages:
    react: ^16.9.0 => 16.12.0 
    react-native: ^0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native: 0.59.10

```

Library version: 5.4.1

## Steps To Reproduce

1. Ensure "yarn" is *not* installed
2. Run "npm i" on a project depending on this library.
...

Describe what you expected to happen:

1. Installation completes successfully

## Reproducible sample code

N/A

--- a/package.json
+++ b/package.json
@@ -20,21 +20,21 @@
     "url": "https://github.com/react-native-community/react-native-device-info"
   },
   "scripts": {
-    "analyze": "yarn ts-check && yarn flow-check",
+    "analyze": "npm run ts-check && npm run flow-check",
     "flow-check": "npx flow-bin check-contents < src/index.js.flow",
     "ts-check": "npx tsc --noEmit",
     "clean": "cd example && npx react-native-clean-project --keep-node-modules --remove-iOS-build --remove-iOS-pods --remove-android-build --keep-brew --keep-pods && \\rm -fr ios/Pods",
-    "dev-sync": "yarn build && cp -r *podspec lib windows android ios src example/node_modules/react-native-device-info/",
+    "dev-sync": "npm run build && cp -r *podspec lib windows android ios src example/node_modules/react-native-device-info/",
     "lint": "npx eslint ./ --ignore-pattern example --ignore-pattern node_modules --fix --quiet",
-    "shipit": "yarn clean && np",
+    "shipit": "npm run clean && np",
     "test": "echo no tests defined",
-    "precommit": "lint-staged && yarn analyze",
+    "precommit": "lint-staged && npm run analyze",
     "build": "bob build",
-    "prepare": "yarn build"
+    "prepare": "bob build"
   },
   "lint-staged": {
     "*.ts": [
-      "yarn lint",
+      "npm run lint",
       "git add"
     ]
   },

